bld/cb.cs

```diff
@@ -491,7 +491,7 @@ IList<string> libs
 		};
 		var arches_device = new string[] {
 			"arm64",
-			"armv7",
+			//"armv7",
```

Comment:
`armv7` doesn't seem to be a valid target for tvOS. It was previously passing only because wrong SDK was used (see below).